src/argus_htmx/templates/htmx/incidents/_incidents_update_menu.html

```diff
@@ -1,7 +1,8 @@
 <ul class="menu menu-horizontal gap-2">
     {% block update_menu_content %}
-        <li><button class="btn btn-accent">Add Ticket</button></li>
-        <li><button class="btn btn-accent">Acknowledge</button></li>
-        <li><button class="btn btn-accent">Close/re-open</button></li>
+        <li>{% include 'htmx/incidents/_incident_ticket_edit_modal.html' with dialog_id="add-ticket-dialog" button_class="btn-accent" button_title="Change ticket" header="Change ticket" explanation="Write an URL of an existing ticket, or nothing to remove existing ticket urls" cancel_text="Cancel" submit_text="Submit" %}</li>
+        <li>{% include 'htmx/incidents/_incident_acknowledge_modal.html' with dialog_id="create-acknowledgment-dialog" button_class="btn-accent" button_title="Acknowledge" header="Submit acknowledgment" explanation="Write a message describing why these incidents were acknowledged" cancel_text="Cancel" submit_text="Submit" %}</li>
+        <li>{% include 'htmx/incidents/_incident_close_modal.html' with dialog_id="close_incident-dialog" button_class="btn-accent" button_title="Close" header="Manually close incidents" explanation="Write a message describing why these incidents were manually closed" cancel_text="Cancel" submit_text="Close now" %}</li>
+        <li>{% include 'htmx/incidents/_incident_reopen_modal.html' with dialog_id="reopen-incident-dialog" button_class="btn-accent" button_title="Reopen" header="Manually reopen incident" explanation="Write a message describing why these incident were manually reopened" cancel_text="Cancel" submit_text="Reopen now" %}</li>
```

Comment:
```suggestion
        <li>{% include 'htmx/incidents/_incident_reopen_modal.html' with dialog_id="reopen-incident-dialog" button_class="btn-accent" button_title="Reopen" header="Manually reopen incidents" explanation="Write a message describing why these incidents were manually reopened" cancel_text="Cancel" submit_text="Reopen now" %}</li>
```

Reply:
Hoho, this is a problem, the templates are also used for the details page (where you only change one incident at a time).

Reply:
We won't need to have separate templates, phew.